Felienne/hedy
# [BUG FIX] Fixes bug where teacher was unable to remove student from class

**Description**
In this PR we fix a bug where teachers were unable to remove a student from their class. For an explanation of the bug and the solution, take a look at the issue itself.

**Fix for**
This PR fixes #1854.

**How to test**
Make sure you are a teacher and verify that you can remove a student from a class **that is not your own account**.
